Representable/representable
# Add verified organizations

**changes**
- added field to model for an organization for verified, which defaults to false
- if verified=True, displays a checkmark next to the org's name when it appears on the site

**to test**
- python manage.py runserver
- in postico, change an organization's Verified field to True
- check for check mark where organization name appears

**screenshots**
on org dashboard
![image](https://user-images.githubusercontent.com/41943646/111319794-c1a66780-863c-11eb-9d5a-da061d9f1865.png)
on state map page
![image](https://user-images.githubusercontent.com/41943646/111319877-d420a100-863c-11eb-9642-ef9226d6b49d.png)
on drive landing page
![image](https://user-images.githubusercontent.com/41943646/111319937-e3075380-863c-11eb-84dd-fe88f46531b8.png)
